src/app/editor/page.tsx

```
@@ -31,6 +34,77 @@ export default function Editor() {
+		window.addEventListener('editor:save', handleSaveProject);
+		
+		// Expose save state to window for Navbar access
+		(window as Window & { editorSaveState?: { isSaving: boolean; hasUnsavedChanges: boolean; lastSaved: Date | null } }).editorSaveState = {
+			isSaving,
+			hasUnsavedChanges,
+			lastSaved,
+		};
+
+		return () => {
+			window.removeEventListener('editor:save', handleSaveProject);
+			delete (window as Window & { editorSaveState?: { isSaving: boolean; hasUnsavedChanges: boolean; lastSaved: Date | null } }).editorSaveState;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)